package.json

```diff
@@ -8,39 +8,37 @@
       "addons/*",
       "ui/*",
       "e2e-tests"
-    ],
-    "nohoist": [
-      "**/electron**"
     ]
   },
   "scripts": {
     "build": "yarn build:ui:admin",
-    "build:ui:admin": "yarn --cwd ui/admin build",
-    "build:ui:admin:oss": "yarn --cwd ui/admin build:oss",
-    "build:ui:admin:enterprise": "yarn --cwd ui/admin build:enterprise",
-    "build:ui:admin:hcp": "yarn --cwd ui/admin build:hcp",
-    "build:ui:desktop": "yarn --cwd ui/desktop build",
-    "build:ui:desktop:production": "yarn --cwd ui/desktop build:production",
-    "build:ui:desktop:app": "yarn --cwd ui/desktop build:desktop",
+    "build:ui:admin": "yarn workspace admin build",
```

Comment:
switched these to workspace since we have workspaces support and it seemed like a better fit and supposedly faster than `--cwd`